scripts/build_package.py

```diff
@@ -251,7 +251,7 @@ def handle_external_repository(dep, clean, env, verbose):
         if source:
             tmp = tempfile.mkdtemp()
             LOG.debug("Downloading font files " + source + " to " + tmp)
-            for fformat, agent in font_user_agents.iteritems():
+            for fformat, agent in font_user_agents.items():
```

Comment:
I think since this collection should not be large, it is not a performance critical part of the code. I value forward compatibility in this case over performance.